### PR TITLE
Updates Readme with Model checkpointing section

### PR DESCRIPTION
**Pull request by Adamits**
Addresses #6

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -79,6 +79,15 @@ Alternatively, for the SIGMORPHON 2016 shared task data format:
 
 this format is specified by `--features-col 2 --features-sep , --target-col 3`.
 
+## Model checkpointing
+
+Checkpointing is largely handled by [Lightning](https://pytorch-lightning.readthedocs.io/en/stable/common/checkpointing_basic.html). The path for model information, including checkpoints, is specified by a combination of 
+`--model_dir` and `--experiment`, such that we build the path `model_dir/experiment_name/version_*n*`, where each run of an experiment with the same model_dir and experiment_name is namespaced with a new version number. Within a version, we store everything needed to reload a model, namely the hyperparameters (`model_dir/experiment_name/version_*n*/harapams.yaml`), and a directory of checkpoints (`model_dir/experiment_name/version_*n*/checkpoints`).
```
Comment (kylebgorman) [inline README.md:85]: typo: "harapams".

Is your intention for the n to be bolded or italiced here? Unfortunately that does not seem to work in GitHub's markdown formulation.

File: README.md [hunk 2]
```diff
@@ -79,6 +79,15 @@ Alternatively, for the SIGMORPHON 2016 shared task data format:
...
+
+By default, each run initializes a new model from scratch, unless the `--train_from` argument is specified. To continue training from a specific checkpoint, the **full path to the checkpoint** should be specified with for the `--train_from` argument. This creates a new version, but starts from the provided model checkpoint.
+
+During training, we save the best `--save_top_k` checkpoints according to accuracy on the `--dev` set. For example, `--save_top_k 5` will save the top 5 most accurate models, according to the evaluation during training.
```
Comment (kylebgorman) [inline README.md:89]: This is good but maybe you want to say the default (I think it's 1). E.g.:

"...we save the best `--save_top_k` checkpoints (by default, 1) ranked according to accuray on the `--dev` set."

**Review by kylebgorman — COMMENTED** [review]
A few nits but looks great, thanks. You can mark this as closing #6 if you like.

File: README.md (hunk 1 above)
Comment (Adamits) [reply README.md:85]: Yeah, oops I should have double checked when I rendered it. Will remove.

**Comment by kylebgorman** [comment]
BTW not required here (I can do it myself) but I use the following to reflow Markdown:

```
alias reflow='pandoc -f markdown -t markdown --columns=80'
# then
reflow doc.md > fixed.md
```

**Comment by Adamits** [comment]
> BTW not required here (I can do it myself) but I use the following to reflow Markdown:
> 
> ```
> alias reflow='pandoc -f markdown -t markdown --columns=80'
> # then
> reflow doc.md > fixed.md
> ```

Oh cool! Thanks.

**Review by kylebgorman — APPROVED** [review]
LGTM

**Pull request merged**